### DISCUSSION TITLE
Hythens in variables

### DISCUSSION THREAD

**Discussion by sammypanda** [savonet/liquidsoap#4260] [Q&A] [answered]
hai!

I have this data to parse, that im fetching from a URL.

```py
let json.parse ({
        subsonic-response
    } : {
        subsonic-repsonse: {
            status: string
            version: string
        }
})

print("blah: #{subsonic-response}")
```

and im not sure how to handle it when there is a hythen in the object key

```md
INFO: Loading Sdl_image, Target = linux
At script.liq, line 30, char 16:
        subsonic-response

Error 2: Parse error
```

How can i access/use ``subsonic-response`` from my JSON object, when the hythen isn't allowed?

**Comment by toots** [comment, marked as answer]
Hi! For JSON keys that are not representable as variables, you have the `"weird name" as weird_name` syntax as documented at https://www.liquidsoap.info/doc-2.3.0/json.html#objects:

```liquidsoap
let json.parse ({
        subsonic_response
    } : {
        "subsonic-repsonse" as subsonic_response: {
            status: string
            version: string
        }
})
```

**Reply by sammypanda** [reply]
oh okay, thank you so much for answering :)
